### PR TITLE
Update Leaderboard

### PR DESCRIPTION
**Pull request by nkaenzig**
- Replace PANDA & Camelyon16 by PANDASmall & Camelyon16Small
- Added hibou, gigapath & bioptimus models

### CODE REVIEW

File: tools/generate_leaderboard_plot.py [hunk 1]
```diff
@@ -34,14 +34,17 @@ def main():
         "dino_vitb16_kaiko": "kaiko.ai - DINO ViT-B16 | TCGA",
         "dino_vitb8_kaiko": "kaiko.ai - DINO ViT-B8 | TCGA",
         "dino_vitl14_kaiko": "kaiko.ai - DINOv2 ViT-L14 | TCGA",
+        "bioptimus_h_optimus_0": "Bioptimus - H-optimus-0 | ViT-G14",
+        "prov_gigapath": "Prov-GigaPath | DINOv2 ViT-G14",
+        "histai_hibou_l": "hist.ai - hibou-L | DINOv2 ViT-B14",
```
Comment (roman807) [inline tools/generate_leaderboard_plot.py:39]: for the other FMs we introduced the syntax "<model> | <training data>". Lets keep this pattern for the three additional FMs?

File: docs/images/leaderboard.svg
Comment (roman807) [inline docs/images/leaderboard.svg:1]: Lets update this with the latest Cam16 run & updated hibou-L

File: tools/generate_leaderboard_plot.py (hunk 1 above)
Comment (nkaenzig) [reply tools/generate_leaderboard_plot.py]: done

File: docs/images/leaderboard.svg
Comment (nkaenzig) [reply docs/images/leaderboard.svg:1]: done

**Review by roman807 — APPROVED** [review]

**Pull request merged**